wiki/Help_centre/Beatmapping/id.md

```diff
@@ -41,9 +41,9 @@ Apabila ternyata isi kolom tersebut tidak sesuai, tulis nama penggunamu di sana
 
 Apabila langkah-langkah di atas tidak menyelesaikan masalahmu, harap baca [laman Panduan mengatasi masalah-masalah BSS berikut](/wiki/Guides/BSS_Issues) dengan seksama.
 
-## Saya memiliki osu!supporter yang aktif, namun saya tidak dapat mengunggah hingga 10 beatmap Pending/WIP sebagaimana yang dijanjikan!
+## Saya memiliki osu!supporter yang aktif, namun saya tidak dapat mengunggah hingga 20 beatmap Pending/WIP sebagaimana yang dijanjikan!
 
-**Jumlah beatmap Pending/WIP yang dapat kamu miliki dalam satu waktu tergantung pada jumlah beatmap Ranked yang kamu miliki.**
+**Jumlah beatmap Pending/WIP yang dapat kamu miliki dalam satu waktu tergantung pada jumlah beatmap Ranked yang kamu miliki. For details, see [osu!supporter § Increased limits](/wiki/osu!supporter#increased-limits).**<!-- FIXME: replace the last sentence with actual translation -->
```

Comment:
```suggestion
**Jumlah beatmap Pending/WIP yang dapat kamu miliki dalam satu waktu tergantung pada jumlah beatmap Ranked yang kamu miliki. Untuk informasi selengkapnya, kunjungi [osu!supporter § Increased limits](/wiki/osu!supporter#increased-limits).**<!-- FIXME: replace the last sentence with actual translation -->
```

Reply:
the page is still using english since the target page (osu! supporter) is outdated.